spec/integration/guest_view_spec.rb

```diff
@@ -84,6 +85,14 @@
     end
   end
 
+  describe "prohibited paths" do
+      it "redirects the guest to the root when attempting to visit orders_path" do
+      visit orders_path
+
+      expect(current_path).to eq(root_path)
+    end
```

Comment:
`end` at 93, 4 is not aligned with `it "redirects the guest to the root when attempting to visit orders_path" do` at 89, 6